ops/terraform/env/prod/stateful/main.tf

```diff
@@ -16,6 +16,13 @@ module "stateful" {
     allocated_storage = 16000
   }
 
+  db_params = [
+    {name="max_worker_processes", value="16", apply_on_reboot=true},
+    {name="random_page_cost", value="1", apply_on_reboot=false},
+    {name="temp_buffers", value="8192", apply_on_reboot=false},
+    {name="work_mem", value="32768", apply_on_reboot=false}
+  ]
+
   db_import_mode = {
```

Comment:
Cool way to do this.

Reply:
Also: I don't think there's any reason to only apply these changes in prod; as long as our DB HW is the same in all environments (it is now), we likely want the same config in all environments.

Reply:
> Also: I don't think there's any reason to only apply these changes in prod; as long as our DB HW is the same in all environments (it is now), we likely want the same config in all environments.

RDS instances in prod-sbx (db.m5.2xlarge) are a good bit smaller than in test or prod (db.r5.24xlarge)